api/src/main/java/org/pmiops/workbench/google/DirectoryService.java

```diff
@@ -1,25 +1,29 @@
 package org.pmiops.workbench.google;
```

Comment:
Changes in this file were mostly opportunistic (e.g. s/emailAddress/email/ in method names), but the key need was to add a method supportting conversion from the full GSuite email address (gjordan@fake-research-aou.org) to the username (gjordan).